tools/getVersions.ts

```diff
@@ -0,0 +1,11 @@
+import request from "./request";
+
+async function getVersions(ofEnd:'frontend'|'backend') {
+    if(ofEnd === 'frontend') {
+        return useRuntimeConfig().public.FRONTEND_APP_VERSION
+    } else if(ofEnd === 'backend') {
+        return (await request('/version')).version
```

Comment:
This might be change in the feature, we want to return version for backend, vector-db and llamacpp. We can support this in the next PR.

Reply:
okay I do got some ideas, depending the APIs